examples/gno.land/p/demo/memeland/memeland.gno

```diff
@@ -113,7 +118,7 @@ func (m *Memeland) GetPostsInRange(startTimestamp, endTimestamp int64, page, pag
 			},
 		}
 		sort.Sort(dateSorter)
-	default:
+	case "DATE_CREATED":
```

Comment:
Why not define these as constants?

Reply:
Defined: [015902f](https://github.com/gnolang/gno/pull/1817/commits/015902f694b808371b3cfc3494a5423c465b5cfa)